src/NosCore.GameObject/Character.cs

```diff
@@ -860,5 +862,15 @@ public TalkPacket GenerateTalk(string message)
                 Message = message
             };
         }
+
+        public ClPacket GenerateInvisible()
```

Comment:
This should be in the VisualEntityExtension I think, also, Invisible & InvisibleGm Properties might have to be set in the ICharacterEntity ;)

Reply:
Definitely not in the extension visual entity but it could be a extension of ICharacter ;) there is no visual type so it’s a character packet. Could eventually be added to the interface ICharacterEntity